cumulusci/tasks/bulkdata/load.py

```diff
@@ -631,3 +639,27 @@ def _generate_contact_id_map_for_person_accounts(
 
                 # Join maps together to get tuple (Contact ID, Contact SF ID) to insert into step's ID Table.
                 yield (contact_id, contact_sf_id)
+
+    def _set_viewed(self):
+        """Set items as recently viewed. Filter out custom objects without custom tabs."""
+        object_names = set()
+        custom_objects = set()
+
+        # Separate standard and custom objects
+        for mapping in self.mapping.values():
+            object_name = mapping.sf_object
+            if object_name.endswith("__c"):
+                custom_objects.add(object_name)
+            else:
+                object_names.add(object_name)
+        # collect SobjectName that have custom tabs
+        if custom_objects:
+            for record in self.sf.query_all(
+                "SELECT SObjectName FROM TabDefinition WHERE IsCustom = true AND SObjectName IN ('{}')".format(
+                    "','".join(sorted(custom_objects))
+                )
+            )["records"]:
+                object_names.add(record["SobjectName"])
+
+        for mapped_item in sorted(object_names):
+            self.sf.query_all(f"SELECT Id FROM {mapped_item} LIMIT 1000 FOR VIEW")
```

Comment:
There's nothing here to make sure that we're specifically selecting records that were just inserted, is there? Should we do a reverse sort by creation date? Should we limit it to the number of items that were inserted?